cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -288,14 +300,19 @@ func generateMountsModeRun() error {
 			// check for the try kernel
 			tryKernel, tryKernelExists, err := ebl.TryKernel()
 			// TODO:UC20: can we log somewhere if err != nil here?
-			if tryKernelExists && err == nil {
-				kernel = tryKernel
+			// TODO:UC20: can we log somewhere if this kernel snap isn't in the
+			//            list of trusted kernel snaps?
+			tryKernelFile := tryKernel.Filename()
```

Comment:
this might fail if err != nil, no?

can tryKernelExitsts be true but err != nil though ?

Reply:
Yes you're right, I will fix this

Reply:
Done